### PR TITLE
Fix GA analytics pageview

### PR DESCRIPTION
**Pull request by chrishanes**

### CODE REVIEW

File: app/assets/javascripts/modules/moj.AsyncGA.js [hunk 1]
```diff
@@ -13,9 +13,9 @@
       window.ga('create', gaTrackingId, 'service.gov.uk');
 
       if (hitTypePage) {
-        window.ga('send', 'page_view', location.pathname + '#' + hitTypePage);
+        window.ga('send', 'pageview', location.pathname + '#' + hitTypePage);
```
Comment (StupidCodeFactory) [inline app/assets/javascripts/modules/moj.AsyncGA.js]: ?? huh that's odd I remember fixing that a little while ago. Could we add test around this before it breaks again?

Comment (chrishanes) [inline app/assets/javascripts/modules/moj.AsyncGA.js]: I think you fixed on Public

Comment (StupidCodeFactory) [inline app/assets/javascripts/modules/moj.AsyncGA.js]: Ah :) that would make sense yeah

Comment (chrishanes) [inline app/assets/javascripts/modules/moj.AsyncGA.js]: Can you approve now? Not sure we need a test (can't see this file being looked at again)?

**Pull request merged**